src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -328,7 +328,7 @@ public static void updateGeneratorVoltageControl(GeneratorVoltageControl voltage
         checkNotDependentVoltageControl(voltageControl);
         LfBus controlledBus = voltageControl.getControlledBus();
         if (voltageControl.isLocalControl()) {
-            if (voltageControl.isHidden()) {
+            if (voltageControl.isHidden()) { // FIXME
```

Comment:
Need to discuss with you about what does it means for generator voltage control which priority is 0.